course/sis_test.py

```diff
@@ -38,19 +43,31 @@ class SisTest(StudentInfoSystem):
 
     SIS_PLUGIN_MODULE = 'course.sis_test'
     SIS_PLUGIN_CLASS = 'SisTest'
+
+    By using optional SIS_TEST_FILE setting, one can specify a file from which the SIS
+    configuration is read, overriding the default setup above.
     '''
 
+    def __init__(self):
+        self.data = origdata
+        if hasattr(settings, 'SIS_TEST_FILE'):
+            try:
+                f = open(settings.SIS_TEST_FILE, "r")
+                self.data = json.load(f)
```

Comment:
Files should be closed too. The `with` block takes care of closing automatically.
```suggestion
                with open(settings.SIS_TEST_FILE, "r") as f:
                    self.data = json.load(f)
```